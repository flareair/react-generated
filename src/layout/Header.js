import React from 'react';
import PropTypes from 'prop-types';

export default function Header(props) {
	return (
	  <header>
      <h1>{props.appName}</h1>
    </header>
  );
}

Header.propTypes = {
	appName: PropTypes.string
};
