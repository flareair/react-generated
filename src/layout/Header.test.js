import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';


describe('Header component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Header appName={'testApp'}/>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
