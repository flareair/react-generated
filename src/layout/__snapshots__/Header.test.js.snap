// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders correctly 1`] = `
<header>
  <h1>
    testApp
  </h1>
</header>
`;
